src/memory-analyzer/memory_analyzer_main.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 Author: Malte Mues <mail.mues@gmail.com>
```

Comment:
All of these files should have their headers changed to our format

Reply:
Done, hopefully everywhere.